.github/workflows/CI-BUILD.yml

```
@@ -235,14 +235,14 @@ jobs:
       - name: "Attest Build Checksums"
         id: multicast-build-chksum-attest
         if: ${{ !cancelled() && (github.repository == 'reactive-firewall-org/multicast') && (needs.BUILD.envs.ENVIRONMENT == 'Deployment' && startsWith(github.ref, 'refs/tags/v')) }}
```

actionlint (1.7.7)

property "envs" is not defined in object type {outputs: {artifact-digest: string; artifact-files: string; artifact-id: string; artifact-url: string; build_status: string}; result: string}

(expression)